infrastructure-damaged.json

```diff
@@ -0,0 +1,4015 @@
+[
+  {
+    "report_date": "2023-10-07",
+    "ext_civic_buildings": {
+      "destroyed": 5
+    },
+    "ext_educational_buildings": {
+      "destroyed": -2,
+      "damaged": 15
+    },
+    "ext_mosques": {
+      "destroyed": -6,
+      "damaged": -13
+    },
+    "ext_churches": {
+      "destroyed": 0
+    },
+    "ext_units": {
+      "destroyed": 80
+    }
+  },
+  {
+    "report_date": "2023-10-08",
+    "ext_civic_buildings": {
+      "destroyed": 11
+    },
+    "ext_educational_buildings": {
+      "destroyed": -1,
+      "damaged": 30
+    },
+    "ext_mosques": {
+      "destroyed": -4,
+      "damaged": -9
+    },
+    "ext_churches": {
+      "destroyed": 0
+    },
+    "units": {
+      "destroyed": 159
+    },
+    "ext_units": {
+      "destroyed": 159
+    }
+  },
+  {
+    "report_date": "2023-10-09",
+    "ext_civic_buildings": {
+      "destroyed": 16
+    },
+    "ext_educational_buildings": {
+      "destroyed": -1,
+      "damaged": 45
+    },
+    "ext_mosques": {
+      "destroyed": -2,
+      "damaged": -4
+    },
+    "ext_churches": {
+      "destroyed": 0
+    },
+    "units": {
+      "destroyed": 790
+    },
+    "ext_units": {
+      "destroyed": 790
+    }
+  },
+  {
+    "report_date": "2023-10-10",
+    "ext_civic_buildings": {
+      "destroyed": 22
+    },
+    "ext_educational_buildings": {
+      "destroyed": 0,
+      "damaged": 60
+    },
+    "ext_mosques": {
+      "destroyed": 0,
+      "damaged": 0
+    },
+    "ext_churches": {
+      "destroyed": 0
+    },
+    "units": {
+      "destroyed": 1009
+    },
+    "ext_units": {
+      "destroyed": 1009
+    }
+  },
+  {
+    "report_date": "2023-10-11",
+    "civic_buildings": {
+      "destroyed": 27
+    },
+    "ext_civic_buildings": {
+      "destroyed": 27
+    },
+    "educational_buildings": {
+      "destroyed": 3,
+      "damaged": 75
+    },
+    "ext_educational_buildings": {
+      "destroyed": 3,
+      "damaged": 75
+    },
+    "mosques": {
+      "destroyed": 10
+    },
+    "ext_mosques": {
+      "destroyed": 10,
+      "damaged": 4
+    },
+    "ext_churches": {
+      "destroyed": 0
+    },
+    "units": {
+      "destroyed": 2835
+    },
+    "ext_units": {
+      "destroyed": 2835
+    }
+  },
+  {
+    "report_date": "2023-10-12",
+    "ext_civic_buildings": {
+      "destroyed": 19
```

Comment:
number dropped from 27 in prior report? i wouldn't expect any numbers to decrease at this point